src/LinuxTracing/LinuxTracingIntegrationTest.cpp

```diff
@@ -155,7 +156,19 @@ class BufferTracerListener : public TracerListener {
   }
 
   void OnCallstackSample(orbit_grpc_protos::CallstackSample callstack_sample) override {
+    static uint64_t next_callstack_id = 1;
+
+    // TODO: Maybe test it on the sender level?
```

Comment:
What does this TODO mean? I would make this clear and possibly create a bug, or actually do it, or just remove it altogether.

Reply:
removed